dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -65,3 +68,29 @@ module rpCosmosDb '../modules/rp-cosmos.bicep' =
   }
 
 output frontend_mi_client_id string = frontendMI.uamiClientID
+
+module maestroInfra '../modules/maestro-infra.bicep' = if (deployMaestroInfra) {
+  name: 'maestro-infra'
+  scope: resourceGroup()
+  params: {
+    currentUserId: currentUserId
+    location: location
+  }
+}
+
+module maestroServer '../modules/maestro-eventgrid-access.bicep' = if (deployMaestroInfra) {
+  name: 'maestro-server-eventgrid-access'
+  scope: resourceGroup()
+  params: {
+    eventGridNamespaceName: maestroInfra.outputs.eventGridNamespaceName
+    keyVaultName: maestroInfra.outputs.keyVaultName
+    keyVaultManagedIdentityId: maestroInfra.outputs.keyVaultCertificateOfficerManagedIdentityId
+    certDomain: 'maestro.${location}.azure.com'
```

Comment:
consider this a placeholder. the details about the oncert domains to use will be determined in the DDR